src/app/content/components/Page/PageComponent.tsx

```diff
@@ -47,6 +48,14 @@ export default class PageComponent extends Component<PagePropTypes> {
 
       transformContent(parsedContent, parsedContent.body, this.props.intl);
 
+      if (this.props.inAssigned) {
+        const links = parsedContent.body.querySelectorAll('a[href]');
```

Comment:
it shouldn't be all links, but all links to another page. a link to an anchor on the same page is ok.

Reply:
well thats not good